aries-site/src/pages/components/textarea.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Meta, SubsectionText } from '../../components';
+import { ContentSection, Example, Layout, Subsection } from '../../layouts';
+import { TextAreaExample } from '../../examples';
+import { getPageDetails } from '../../utils';
+
+const title = 'TextArea';
+const page = getPageDetails(title);
+const topic = 'Components';
+
+const TextArea = () => (
+  <Layout title={title}>
+    <Meta
+      title={title}
+      description={page.seoDescription}
+      canonicalUrl="https://design-system.hpe.design/components/textarea"
+    />
+    <ContentSection>
+      <Subsection name={title} level={1} topic={topic}>
+        <SubsectionText>
+        When you need to allow the user to provide longer forms of
```

Comment:
This doesn't need to be incorporated into this PR, but I am propose that the first piece of text after the page header be `page.description` which should be a short, to the point description/definition. Any extended description should be moved to the "Show Details" portion of the page. Thoughts?

Reply:
I agree I saw `Anchor` using both the Subsection text with the details we should discuss this more next week